--- /etc/ferm/ferm.conf	2016-01-19 15:16:05.944893150 +0100
+++ /tmp/ferm.cfg	2016-01-29 10:09:47.203740564 +0100
@@ -70,7 +70,15 @@
             #mod state state INVALID DROP;
             mod state state (ESTABLISHED RELATED) ACCEPT;
         }
-     }
+    }
+    table nat {
+        chain PREROUTING {
+            policy ACCEPT;
+
+            proto (udp tcp) dport 80 REDIRECT to-ports 3478;
+            proto (udp tcp) dport 443 REDIRECT to-ports 5349;
+        }
+    }
 }
 
 #IPv6:
@@ -141,5 +149,13 @@
             #mod state state INVALID DROP;
             mod state state (ESTABLISHED RELATED) ACCEPT;
         }
-     }
+    }
+    table nat {
+        chain PREROUTING {
+            policy ACCEPT;
+
+            proto (udp tcp) dport 80 REDIRECT to-ports 3478;
+            proto (udp tcp) dport 443 REDIRECT to-ports 5349;
+        }
+    }
 }
